.gitignore

```diff
@@ -9,10 +9,8 @@
 /coverage
 
```

Comment:
Please add these as well:

```
# Optional npm cache directory
.npm
.pnpm-store
```

Reply:
When you run `pnpm install`, it creates over 50.000 files as cache. This adds them to .gitignore.